external/anomaly/adapters/anomalib/callbacks/score_report.py

```diff
@@ -41,4 +42,10 @@ def on_validation_epoch_end(self, trainer, pl_module):  # pylint: disable=unused
                     score = score + int(trainer.global_step)
                 else:
                     score = -(score + int(trainer.global_step))
-            self.score_reporting_callback(progress=0, score=score)
+
+            # Always assumes that hpo validation step is called during training.
+            training_progress = (
+                (self.train_batch_idx + trainer.current_epoch * self.total_train_batches)
+                / (self.total_train_batches * trainer.max_epochs)
```

Comment:
maybe // instead of /, so we could get rid of int(training_progress) in L51

Reply:
Not sure if it will work though. We get a decimal value in [0,1] which we multiply with 100 to get the percentage.

Reply:
yeah true, never mind that